code/mob/living/critter/flock/flockdrone.dm

```diff
@@ -1145,7 +1111,7 @@
 /datum/equipmentHolder/flockAbsorption/on_unequip()
 	var/obj/item/temp = item
 	if(temp)
-		animate(temp) // cancel animation
```

Comment:
useful comment

Reply:
I removed it as I think animate(...) is used in a lot of places with no comment

Reply:
It is, but it ought not to be imo